modules/space/rendering/renderablestars.cpp

```diff
@@ -309,120 +305,71 @@ namespace {
         "Disable Fade-in effect",
         "Enables/Disables the Fade-in effect."
     };
+
+    struct [[codegen::Dictionary(RenderableStars)]] Parameters {
+        // The path to the SPECK file containing information about the stars being rendered
+        std::string file;
```

Comment:
Should it be `file` or `speckFile`? 

In the PropertyInfo, it is given as "SpeckFile", but maybe that one should be changed.

Reply:
I renamed the variable for now, but left the Key to "File".  I think it might unnecessarily break people if we change it to `SpeckFile`.  Maybe a bit later?